.travis.yml

```diff
@@ -48,4 +51,4 @@ after_script:
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
 
 services:
-  rabbitmq
+  rabbitmq
```

Comment:
Travis does not support rabbitmq out of the box, it must be installed using apt packages.

Reply:
ah, sorry, it is just line ending..